WordPress/gutenberg
# Remove label from dataview checkbox

## What?
Fixes https://github.com/WordPress/gutenberg/issues/59178

This PR updates the labels associated with checkboxes in data views to remove the dynamically generated state values (Select item: and Deselect item:). The labels now display only the relevant item title, simplifying and improving accessibility.

## Why?
The current implementation of checkbox labels includes dynamic state indicators (Select item: and Deselect item:) which
 overcomplicate the labels and do not adhere to the semantic purpose of labels as descriptors of the control.

## How?
Removed the dynamic prefixes (Select item: and Deselect item:) from the checkbox labels in the code. Updated the label to display only the item's title, derived from the provided titleField property.

## Testing Instructions

- Navigate to the Site Editor > Pages > Manage all pages.
- Hover over a table row to make the checkbox visible.
- Verify that the label for each checkbox contains only the item's title.
- Select and deselect a checkbox.
- Confirm that the label text remains unchanged and only displays the item title.

## Screenshots or screencast 

https://github.com/user-attachments/assets/16aa9b1e-af77-4de5-aec5-38f5eb70fb36
